tests/src/python/utilities.py

```diff
@@ -537,10 +537,8 @@ def parseClassElem(self, e):
         # test for brief description
         d = e.find('briefdescription')
         has_brief_description = False
-        if d:
-            p = d.find('para')
-            if p.text and len(p.text) > 0:
-                has_brief_description = True
+        if d and d.find('para'):
```

Comment:
@nyalldawson I hope that doesn't hide any alarms.

That's what didn't work before:

    <briefdescription>
        <para><ref kindref="compound" refid="classQgsOptional">QgsOptional</ref> is a container for other classes and adds an additional enabled/disabled flag. </para>
    </briefdescription>

I checked with QgsSurface which doesn't include any brief description and it should still trigger an error there.